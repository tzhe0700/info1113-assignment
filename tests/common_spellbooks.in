ADD STUDENT Harry Potter
ADD STUDENT Hermione Granger
ADD COLLECTION spellbooks.csv
RENT 100000 111111
RENT 100000 111112
RELINQUISH ALL 100000
RENT 100001 111111
RENT 100001 111112
RELINQUISH ALL 100001
COMMON 100000 100001
EXIT

