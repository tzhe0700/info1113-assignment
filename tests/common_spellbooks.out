user: Success.

user: Success.

user: 1 spellbooks successfully added.

user: Success.

user: No such spellbook in system.

user: Success.

user: Success.

user: No such spellbook in system.

user: Success.

user: Advanced Potion Making (Libatius Borage)

user: Ending Archive process.

